rusoto/services/sts/src/lib.rs

```diff
@@ -1,7 +1,45 @@
 
 //! AWS Security Token Service
 //!
-//! If you're using the service, you're probably looking for [StsClient](struct.StsClient.html) and [Sts](trait.Sts.html).
+//! If you're using the service, you're probably looking for [StsClient](struct.StsClient.html), [Sts](trait.Sts.html)
+//! and [StsAssumeRoleSessionCredentialsProvider](struct.StsAssumeRoleSessionCredentialsProvider.html).
```

Comment:
Unfortunately, changes to this file will be removed when services are regenerated (we probably want to add a warning comment to generated files regarding that). At the current moment, there isn't really a great way to support this. We'd need to build in a custom documentation part of the service crate generator.